optimum/habana/sentence_transformers/st_gaudi_transformer_tokenize.py

```diff
@@ -1,5 +1,20 @@
+import json
+import os
 from typing import Dict, List, Tuple, Union
 
+import torch
+
+from optimum.habana.utils import to_device_dtype
```

Comment:
@nngokhale above comment! Please put the new function in a new file with the correct name or create a new file and put both functions there

Reply:
Done